src/components/Checkbox/index.jsx

```diff
@@ -3,13 +3,14 @@ import PropTypes from 'prop-types';
 
 import './Checkbox.scss';
 
-const Checkbox = ({ label, name }) => {
+const Checkbox = ({ label, labelSmall, name, className }) => {
   const [checked, setChecked] = useState(false);
 
   return (
-    <div className='checkbox'>
+    <div className={`checkbox ${className}`}>
       <label className='checkbox__label'>
-        {label}
+        <span className='checkbox__label--large'>{label}</span>
+        <span className='checkbox__label--mobile'>{labelSmall}</span>
```

Comment:
Checkbox should receive only one label. 
And parent component should be responsible for what that label will be. 
So create a function for that in `SearchBar` and remove `lableSmall` from here.